tests/pipeline_operations_test.py

```diff
@@ -1,12 +1,47 @@
 import unittest
+import pyspark
 
 import pipeline_dp
 
+
 class PipelineOperationsTest(unittest.TestCase):
   pass
 
+
+class SparkRDDOperationsTest(unittest.TestCase):
+  @classmethod
+  def setUpClass(cls):
+    conf = pyspark.SparkConf()
+    SparkRDDOperationsTest.sc = pyspark.SparkContext(conf=conf)
```

Comment:
Nit (Nit = optional improvement): SparkRDDOperationsTest -> cls

Reply:
Yeah, it'll look better. Changed.